### PR TITLE
biblatex import: allow commands without braces

### PR DESCRIPTION
**Pull request by johanneswilm**

### CODE REVIEW

**Comment by retorquere** [comment]
this doesn't implement `itshape`, `bfseries` and `schape` I think.

**Comment by johanneswilm** [comment]
@retorquere I think this should cover it. This should also cover cases like ``{\it\bf Fish}``

File: src/import/literal-parser.js [hunk 1]
```diff
@@ -120,6 +126,22 @@ export class BibLatexLiteralParser {
...
+                            this.string.substring(this.si, this.si + command[0].length) === command[0] &&
+                            [' ', '\\'].includes(this.string.substring(this.si + command[0].length, this.si + command[0].length + 1))
+                        ) {
+                            this.si += command[0].length
+                            if (this.string.substring(this.si, this.si + 1) === ' ') {
+                                this.si++
+                                console.log('removed space')
+                            }
+                            this.currentMarks.push({type:command[1]})
+                            console.log(this.currentMarks)
```
Comment (retorquere) [inline src/import/literal-parser.js]: shouldn't this be removed?

**Comment by johanneswilm** [comment]
@retorquere Ok, but those could be added to the list, right?

File: src/import/literal-parser.js (hunk 1 above)
Comment (johanneswilm) [reply src/import/literal-parser.js]: yes :)

**Comment by retorquere** [comment]
Seems to me that should work yes

**Comment by retorquere** [comment]
Looks good to me.

**Comment by retorquere** [comment]
How do we cut a new release?

**Comment by retorquere** [comment]
Is it

```
npm run prepare
npm version patch
git push --follow-tags
```
?

**Comment by johanneswilm** [comment]
I don't remember. :)

I think you released the last few times. It might also be that we made it release through travis.

**Comment by retorquere** [comment]
I really don't recall. I've done a version patch, let's see if something turns up on NPM 🤷‍♂

**Comment by retorquere** [comment]
Yeah, looks like that worked.

**Comment by retorquere** [comment]
Thanks!

**Comment by johanneswilm** [comment]
thank you for finding this!

**Comment by retorquere** [comment]
I stumbled upon this when I was working on my own sentence caser -- maybe it's something that might be interesting to work into the parser, but it's pretty tricky (and mine is pretty dumb). But bib(la)tex expects English references to be in Title Case, and CSL expects everything to be in Sentence case. It's one of the more tricky parts of BBT, converting between the two.

**Pull request merged**